source/runtime.hpp

```diff
@@ -208,6 +208,7 @@ namespace reshade
 		void save_texture(const texture &texture);
 		void update_texture(texture &texture, uint32_t width, uint32_t height, uint32_t depth, const void *pixels);
 
+		void reset_uniform_value(api::effect_uniform_variable variable);
```

Comment:
This needs to be defined above in the `public` section (where `set_uniform_value_bool` etc. are defined), otherwise the project will fail to compile when `RESHADE_FX` is not defined (due to the `reset_uniform_value` virtual not being implemented then).